htdocs/product/stock/productlot_list.php

```diff
@@ -216,8 +216,7 @@ function init_myfunc()
 $sql.=$hookmanager->resPrint;
 $sql.= " FROM ".MAIN_DB_PREFIX."product_lot as t";
 if (is_array($extrafields->attribute_label) && count($extrafields->attribute_label)) $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."product_lot_extrafields as ef on (t.rowid = ef.fk_object)";
-$sql.= ", ".MAIN_DB_PREFIX."product as p";
-$sql.= " WHERE p.rowid = t.fk_product";
+$sql.= " LEFT JOIN ".MAIN_DB_PREFIX."product as p ON p.rowid = t.fk_product";
```

Comment:
What is goal of this.
If we have data into product_lot and not into product, it means data in database are corrupted and we should delete/clean line in product_lot that are orphelins